tenzir/CMakeLists.txt

```diff
@@ -90,7 +93,7 @@ elseif (${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
       \"Installing: \$ENV{HOME}/Library/LaunchAgents/com.tenzir.tenzir.plist\")
     configure_file(
       \"${CMAKE_CURRENT_SOURCE_DIR}/services/launchd/com.tenzir.tenzir.plist.in\"
-      \"\$ENV{HOME}/Library/LaunchAgents/com.tenzir.tenzir.plist\" @ONLY)"
+      \"${CMAKE_INSTALL_DATADIR}/Library/LaunchAgents/com.tenzir.tenzir.plist\" @ONLY)"
```

Comment:
This will break the launch agent, as it has to be installed in `~/Library/LaunchAgents` (per-user installation) or `/Library/LaunchAgents` (system-wide installation) or `/System/Library/LaunchAgents` (default system-wide installation).

Reply:
Is it possible to create symlinks in those directories instead?

Reply:
I just tried this out, and unfortunately it makes it so the file won't be picked up.

We can also just make this optional in CMake and disable it in CI.